packages/integration-tests/test/standardExitTest.js

```diff
@@ -83,6 +83,15 @@ describe('Standard Exit tests', async () => {
       const exitData = await childChain.getExitData(utxoToExit)
       assert.hasAllKeys(exitData, ['txbytes', 'proof', 'utxo_pos'])
 
+      try {
```

Comment:
Instead of always calling `rootChain.addToken()` and failing if it's been added already, we could call `rootChain.hasToken()` to check first